lib/pp.rb

```diff
@@ -223,7 +223,7 @@ def seplist(list, sep=nil, iter_method=:each) # :yield: element
         else
           sep.call
         end
-        yield(*v)
+        yield(*v, **{})
```

Comment:
A note: for most cases similar to this, one would use `list.each do |v|`, which doesn't have the problem.
Such code would also work here and passes `test_pp.rb`.
But it could be incompatible if `iter_method` yields multiple arguments (`Hash#each_pair` doesn't for `|x|` blocks, it yields one `[k,w]` argument) and there is a caller of `seplist` which uses these multiple arguments.
So this fix seems more compatible :+1: 

I'm still wondering if we can find a more general fix to avoid removing user-supplied empty keyword arguments hashes though.